apps/condo/domains/notification/schema/Message.js

```diff
@@ -124,19 +137,31 @@ const Message = new GQLListSchema('Message', {
         constraints: [
             {
                 type: 'models.CheckConstraint',
-                check: 'Q(user__isnull=False) | Q(phone__isnull=False) | Q(email__isnull=False)',
-                name: 'has_phone_or_email_or_user',
+                check: 'Q(phone__isnull=False) | Q(email__isnull=False) | Q(user__isnull=False) | Q(remoteClient__isnull=False) | Q(deletedAt__isnull=False)',
+                name: 'has_phone_or_email_or_user_or_remoteClient',
             },
             {
                 type: 'models.UniqueConstraint',
                 fields: ['user', 'type', 'uniqKey'],
-                condition: 'Q(deletedAt__isnull=True)',
+                condition: 'Q(deletedAt__isnull=True) & Q(user__isnull=False) & Q(remoteClient__isnull=True)',
```

Comment:
I don't think that uniqKey should check user or remoteClient. If you really need to use user/or remoteClient in that logic you can add it as a part of the uniqKey!